ci/azp-pipeline.yml

```diff
@@ -25,6 +25,8 @@ jobs:
   timeoutInMinutes: 60
   steps:
   - template: install_deps.yml
+  - script: go get golang.org/x/tools/cmd/goimports
```

Comment:
This is pulling from master, something that has been the source of many issues as the formatting rules change fairly regularly.  To avoid problems, we vendor a version[1] in fabric and install it from the tree[2].

[1]: https://github.com/hyperledger/fabric/blob/0833f035234ff881756a179bddba981dcbe398d2/Gopkg.toml#L7
[2]: https://github.com/hyperledger/fabric/blob/0833f035234ff881756a179bddba981dcbe398d2/gotools.mk#L38-L41

Reply:
Will address in a separate PR